pkgs/applications/misc/kanjidraw/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, bash
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "kanjidraw";
+  version = "0.2.3";
+
+  src = fetchFromGitHub {
+    owner = "obfusk";
+    repo = "kanjidraw";
+    rev = "v${version}";
+    sha256 = "03ag8vkbf85qww857ii8hcnn8bh5qa7rsmhka0v9vfxk272ifbyq";
+  };
+
+  propagatedBuildInputs = with python3Packages; [ tkinter ];
+
+  postPatch = ''
+    substituteInPlace Makefile --replace /bin/bash ${bash}/bin/bash
+  '';
+
+  checkPhase = "make test";
+
+  meta = with lib; {
+    description = "Handwritten kanji recognition";
+    longDescription = ''
+      kanjidraw is a simple Python library + GUI for matching (the strokes of a)
+      handwritten kanji against its database.
+
+      You can use the GUI to draw and subsequently select a kanji from the list of
+      probable matches, which will then be copied to the clipboard.
+
+      The database is based on KanjiVG and the algorithms are based on the
+      Kanji draw Android app.
+    '';
+    homepage = "https://github.com/obfusk/kanjidraw";
+    license = with licenses; [
+      agpl3Plus     # code
+      cc-by-sa-30   # data.json
+    ];
+    maintainers = [ maintainers.obfusk ];
+    platforms = platforms.unix;
```

Comment:
```suggestion
```
platforms for buildPythonApplication are smaller than unix.

Reply:
thx. could you point me to some more detail on that?

Reply:
https://github.com/NixOS/nixpkgs/blob/e77fda35583705bb20fcb5f7be9b2cffb7448fe2/pkgs/development/interpreters/python/mk-python-derivation.nix#L177

Reply:
thx

Reply:
I used this b/c for `jiten` someone suggested changing `.linux` to `.unix`. I'll fix that when I update `jiten`.